scripts/ai/AIDriveStrategyUnloadCombine.lua

```diff
@@ -2219,8 +2219,13 @@ function AIDriveStrategyUnloadCombine:onPathfindingDoneBeforeUnloadingOnField(pa
                 -self.fieldUnloadData.xOffset, dz, zOffset, 3, false))
         self:startCourse(course, 1)
     else
-        self:info("Could not find a path to the field unload position!")
-        self.vehicle:stopCurrentAIJob(AIMessageCpErrorNoPathFound.new())
+        -- This lets pathfinder try again with less strict pathfinder instead of just outright going home with its ball
```

Comment:
Please do not change existing logic, revert this change. @schwiti6190 We'll need to review this whole pathfinder retry logic anyway as it does not make a lot of sense as it is now. It'll relax the constraints but at the next try, the situation will most likely be different, a failed pathfinder run takes a long time and in that time the harvester most likely moved .